pkgs/development/tools/misc/stlink/default.nix

```diff
@@ -27,6 +27,11 @@ in stdenv.mkDerivation rec {
     sha256 = "03xypffpbp4imrczbxmq69vgkr7mbp0ps9dk815br5wwlz6vgygl";
   };
 
+  patches = [
+    # https://github.com/stlink-org/stlink/commit/468b1d2daa853b975c33ab69876c486734f2c6a7
+    ./patches/macos_security_framework.patch
```

Comment:
I think nixpkgs prefers to `pkgs.fetchpatch` these remotely instead of adding them to the tree.